wiki/Ranking_criteria/Simplified_ranking_criteria/en.md

```diff
@@ -0,0 +1,197 @@
+  - **Add the song's [genre and language](/wiki/Beatmap/Genre_and_language).**
+  - **Add `featured artist` if the song is in the [Featured Artist catalogue](https://osu.ppy.sh/beatmaps/artists).**
+  - **Other tags must be relevant to the song/map.**
+
+### Title
+
+  - **Songs used for TV must have `(TV Size)`.**
+  - **Replace game version markers with `(Game Ver.)`.**
+  - **Replace short version markers with `(Short Ver.)`.**
+  - **Use `(Cut Ver.)` to indicate an unofficial song cut.**
+  - **Use `(Extended Edit)` to indicate an unofficial song extension.**
+  - **Use `(Sped Up Ver.)` to indicate an unofficial song tempo increase.** For some genres, `(Nightcore Mix)` is an okay alternative.
+
+### Source
+
+- **Use the Source field if the song is from another media source, like a game or a movie.**
+
+## Files
+
+- **Don't use inappropriate songs or backgrounds.** See [song content rules](/wiki/Rules/Song_content_rules) and [visual content considerations](/wiki/Rules/Visual_content_considerations).
+- **Don't include unused files in the map folder.**
+
+### Song
+
+- **Use `.mp3` or `.ogg` file formats.**
+  - At most 192 kbps for `.mp3` files.
+  - At most 208 kbps for `.ogg` files.
+  - At least 128 kbps for any filetype.
+- **One song file must be used for all difficulties.**
+- **Set a consistent preview point for all difficulties.**
+
+### Hitsounds
+
+::: Infobox
+![](./img/delay.png "Hitsound file with delay")
+:::
+
+- **Hitsound files must be 25ms long and use `.wav` or `.ogg` file formats.**
+  - Do not use `.mp3`.
+  - **Exception:** Use [this file](https://up.ppy.sh/files/blank.wav) for silent hitsounds.
+- **Hitsounds must not be [delayed](./img/delay.png).**
+
+### Background
+
+- **Every difficulty must have a background adhering to the following:**
+  - **Minimum width:** 160px
+  - **Minimum height:** 120px
+  - **Maximum width:** 2560px
+  - **Maximum height:** 1440px
+  - **Maximum file size:** 2.5MB
+- **Link the source of the map's background in the description.**
+
+### Video
+
+- **Video must adhere to the following:**
+  - **Maximum video resolution:** 1280x720
+  - **Video encoding:** H.264
+- **Audio track must be removed from video files.**
+
+## Mode-specific
+
+*For full [game mode](/wiki/Game_mode)-specific ranking criteria, see: [osu!](../osu!), [osu!taiko](../osu!taiko), [osu!catch](../osu!catch), and [osu!mania](../osu!mania)*
+
+### osu!
+
+::: Infobox
+![](./img/offscreen.png "Slightly offscreen slider in 1280x960")
+:::
+
+- **No off-screen objects in 4:3 aspect ratios.**
+- **Auto must achieve bonus score on spinners.** They're too short otherwise.
+- **Use at least two combo colors.**
+
+### osu!taiko
+
+- **Adjust the background's vertical offset to fit the playfield.**
+- **Use `Omit first bar line` to remove any unfitting bar lines.**
+- **Avoid kiai flashes.**
+- **On rhythms that are 1/4 or faster, use big notes only at the end of patterns.**
```

Comment:
@Hiviexd tbh it might be time to add this as a guideline in inner oni for the full rc, as very few people can make it work

Reply:
is this relevant to the pr? (i legitimately dont know lol)

Reply:
@pishifat not relevant dw about it
@aceticke could be, tho it'll likely happen through an RC proposal